CHANGELOG.md

```diff
@@ -2409,10 +2409,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* Use red text when pod installation fails 
+* Use red text when pod installation fails
```

Comment:
this one actually needs two empty spaces in the end for markdown formatting to work.